bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/admin/ColumnDescriptorAdapter.java

```diff
@@ -181,12 +184,12 @@ public static GcRule buildGarbageCollectionRule(HColumnDescriptor columnDescript
       if (maxVersions == Integer.MAX_VALUE) {
         return null;
       } else {
-        return maxVersions(maxVersions);
+        return GCRULES.maxVersions(maxVersions);
       }
     }
 
     // minVersions only comes into play with a TTL:
-    GcRule ageRule = maxAge(ttlSeconds);
+    GCRule ageRule = GCRULES.maxAge(org.threeten.bp.Duration.ofSeconds(ttlSeconds));
```

Comment:
Once you remove the unused methods, you should be able to remove the conflicting Duration import and import this one instead